sljit_src/sljitConfigInternal.h

```diff
@@ -37,6 +37,10 @@
 #include <stdlib.h>
 #endif
 
+#ifdef __APPLE__
+#include <libkern/OSCacheControl.h>
```

Comment:
this change was giving me DejaVu, and it is because I had a similar one from long ago to address building in iOS (which can only use this AFAIK in jailbroken devices)

will refactor as it is also now relevant with current HEAD for macOS in Apple Silicon under some constrained circumstances, and might be even worth doing in an independent series as it is less of a "cleanup" change.